.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: 🐛 Bug Report
+about: If something isn't working as expected 🤔.
+
+---
+
+**Current Behavior**
+A clear and concise description of the behavior.
+
+**Input Code**
```

Comment:
shouldn't this be more end-user friendly?  e.g. they probably won't be able to point to the code in question

Reply:
Agree, I'll refactor the text a bit, think about what actually are the usecases etc.